CrossClusterDataSteps.md

```diff
@@ -27,133 +25,53 @@
-``` bash
-MIG_TOKEN_SECRET=`oc get secret | grep target-pvc-migrator-token | head -1 | awk '{print $1}'`
-# Use token value to generate local temp secret
-oc create secret generic x-cluster-test --from-literal=token=`oc get secret ${MIG_TOKEN_SECRET} -o json | jq -r .data.token | base64 -d` --dry-run=client -o json > tmp.secret
-```
-
-5. Login to Source Cluster (klab: namespace jefkel-tools) and create secret
-``` bash
-oc apply -n jefkel-tools -f ./tmp.secret; rm ./tmp.secret
-oc label secret x-cluster-test base=xc-pvc-migrator
-```
-
-6. Build migrator pod (if required)
-``` bash
-oc process -f ./openshift/templates/pvc-migrator-build.yaml | oc apply -f - -n jefkel-tools
-# Confirm image with :latest tag exists and is the recent build.
-oc get is pvc-migrator
-```
-
-7. Deploy templated manifests to create deployment
-``` bash
-oc process -f ./cross-cluster/source-migrator-tmpl.yaml \
-  --param-file=cross-cluster/tester.env \
-  --ignore-unknown-parameters=true | oc apply -f - -n jefkel-tools
-```
-
-8. Connect to source pod and push data
-> NOTE: the `--kubeconfig=...` is required, otherwise you'll likely get unauthorized errors
-
-``` bash
-# rsh to migration pod:
-oc rsh `oc get pods | grep source-pvc-migrator | awk '{print $1}'`
-# set target pod name:
-MIG_POD_TARGET=`oc --kubeconfig=/target/tkube/KUBECONFIG get pods | grep target-pvc-migrator | awk '{print $1}'`
-# use above target pod to run the rsync:
-cd /source; oc rsync --compress=true --kubeconfig=/target/tkube/KUBECONFIG . ${MIG_POD_TARGET}:/target
-```
-
-9. Scale down deployments
-> This allows re-running the data copy at a future time just by scaling up source and destination, and re-running the rsync.
-``` bash
-# login to source cluster
-oc scale --replicas=0 deployment/source-pvc-migrator
-
-# login to target cluster
-oc scale --replicas=0 deployment/target-pvc-migrator
-```
-
-10. Cleanup steps:
-Cleanup of deployed objects can be done with label selectors:
-``` bash
-# Cleanup deployment objects
-oc delete -l base=xc-pvc-migrator deployment,sa,rolebinding,secret
-# Cleanup build details
-oc delete -l app=pvc-migrator build,buildconfig,is
-```
+#### 1. Copy login commands from clusters.  
+
+It is important that you have logged in recently to both clusters from the command line. Go to the web console of each cluster and copy and paste each login command into your terminal.
+
+
+#### 2. Initialization  
+~~~
+./x-cluster-migrate init -p default
+~~~
+Follow the prompts on the screen and it should successfully initialize the project.  
+You will have to navigate to `./cross-cluster/openshift/templates/source-pvc-migrator/source-pvc-migrator-deploy.source.local.param` and `./cross-cluster/openshift/templates/target-pvc-migrator/target-pvc-migrator-deploy.target.local.param` and comment out the line that says `SRC_IMAGE_NAMESPACE` or `DST_IMAGE_NAMESPACE` and change it to the namespace of your `pvc-migrator` build.  
```

Comment:
The scripts should automate this process.  The user should not have to edit anything manually.  Examples of how to automate the update (comment, uncomment, update values) of the param files can be found here; https://github.com/BCDevOps/matomo-openshift/blob/master/openshift/manage#L229.  Examples are also in the `updateMigratorDeplymentParams` function of the `x-cluster-migrate` script.

Reply:
change completed